Weatherman.py

```diff
@@ -0,0 +1,245 @@
+import os
+import sys
+
+
+def weatherMonth(year, month, res):
+    file_name = ''
+    arr = os.listdir('.')
```

Comment:
can you explain why you use `os.listdir('.')`

Reply:
listing the files that has '.' in them